docs/en/engines/table-engines/integrations/redis.md

```diff
@@ -111,9 +112,16 @@ Flush Redis db asynchronously. Also `Truncate` support SYNC mode.
 TRUNCATE TABLE redis_table SYNC;
 ```
 
+Join:
+
+Join with other tables.
+
+```
+SELECT * FROM redis_table JOIN merge_tree_table ON redis_table.key=merge_tree_table.key;
```

Comment:
Isn't it better the other way around (redis table at the right)?

Reply:
Thanks for your advice.